Cargo.toml

```diff
@@ -45,6 +45,7 @@ serde = "~1.0"
 serde_json = "~1.0"
 uuid = {version = "~0.7", features = ["serde"] }
 rustc-serialize = "~0.3"
+nix = "0.14.1"
```

Comment:
I think `nix="~0.14"` should be fine.

Reply:
I've updated the version, but for what's worth, I believe "0.14.1" would allow any 0.14.x: https://doc.rust-lang.org/cargo/reference/specifying-dependencies.html